design/ceph-nfs-ganesha.md

```diff
@@ -156,12 +139,14 @@ The NFS server cluster can be scaled up or down by updating the
 number of the active Ganesha servers in the CRD (using `kubectl edit` or
 modifying the original CRD and running `kubectly apply -f <CRD yaml file>`).
 
-### Dynamically adding, updating, and  removing exports of running Ganesha servers
+### Per-node config files
+After loading the basic ganesha config from inside the container, the node
+will read the rest of its config from an object in RADOS. This allows external
+tools to generate EXPORT definitions for ganesha.
 
-To dynamically update the exports of a Ganesha cluster you will have to update
-its CRD. On updating the CRD, the operator  will update the exports stored in
-the ConfigMaps, and will send DBus signals to the Ganesha servers to refresh
-their exports.
+The object will be named "conf-<metadata.name>.<index>", where metadata.name
+is taken from the CRD and the index is internally generated. It will be stored
+in RADOS.pool and RADOS.namespace from the above CRD.
```

Comment:
Add inline code blocks around the `RADOS.pool` and `RADOS.namespace`, also as written earlier making the `RADOS` seems better in compliance to naming in the other CRDs.